### PR TITLE
Fixed bug in SearchIncidentsV2

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/47893

## Description
Fixed bug in SearchIncidentsV2 script where getting an `id` of type int.

## Screenshots
Screen Shot 2022-04-10 at 17 53 39

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by yaakovpraisler** [comment]
Failed test: https://app.circleci.com/pipelines/github/demisto/content/156115/workflows/b8158adb-a1a8-43a1-9d15-04ac3b0695f4/jobs/504047/tests

File: Packs/CommonScripts/Scripts/SearchIncidentsV2/SearchIncidentsV2.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def search_incidents(args: Dict):   # pragma: no cover
         args.pop('trimevents')
 
     # handle list of ids
-    if args.get('id'):
+    if args.get('id') and isinstance(args.get('id'), str):
```
Comment (daryakoval) [inline Packs/CommonScripts/Scripts/SearchIncidentsV2/SearchIncidentsV2.py:82]: What if this script is a part of Playbook, and the input args['id'] is List[int]? 
I thinks in this case we will need to create comma separated list of ids as well. 
In addition, if args['id'] is int, does the automation `getIncidents` will work or id should be string?

Comment (yaakovpraisler) [reply Packs/CommonScripts/Scripts/SearchIncidentsV2/SearchIncidentsV2.py]: added a generic workaround for all cases (also UTs).

**Review by dantavori — CHANGES_REQUESTED** [review]

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 2]
```diff
@@ -1764,7 +1764,7 @@ def FormatIso8601(t):
     return t.strftime("%Y-%m-%dT%H:%M:%S")
 
 
-def argToList(arg, separator=','):
+def argToList(arg, separator=',', transform=None):
```
Comment (dantavori) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:1767]: please add the transform description to the docstring

**Review by dantavori — APPROVED** [review]

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 3]
```diff
@@ -1774,18 +1774,30 @@ def argToList(arg, separator=','):
        :type separator: ``str``
        :param separator: A string separator to separate the strings, the default is a comma.
 
+       :type transform: ``collable``
```
Comment (daryakoval) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:1777]:
```suggestion
       :type transform: ``callable``
```

**Review by daryakoval — APPROVED** [review]

**Pull request merged**